### PR TITLE
Asintel 1635 public data extract

### PR DESCRIPTION
**Pull request by sarmaevanidfe**

### CODE REVIEW

**Review by uhimabindu — APPROVED** [review]
Approved

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportPublicSectorDataToPL.sql [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+				   ,getdate()
+				   ,0
+
+			  SELECT @LogID=MAX(LogId) FROM Mgmt.Log_Execution_Results
+			  WHERE StoredProcedureName='ImportPublicSectorReportDataToPL'
+			  AND RunId=@RunID
+
+			BEGIN TRANSACTION
+
+				TRUNCATE TABLE [AsData_PL].[PubSector_Report];
```
[the remark this answers is not in the document]
Comment (sarmaevanidfe) [reply src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportPublicSectorDataToPL.sql:37]: PL table needs to be truncate before each load

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportPublicSectorDataToPL.sql [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+					  ,[Questions_AnythingElse_Id]
+					  ,[Questions_Challenges_Answer]
+					  ,[Questions_Challenges_Id]
+					  ,[Questions_OutlineActions_Answer]
+					  ,[Questions_OutlineActions_Id]
+					  ,[Questions_TargetPlans_Answer]
+					  ,[Questions_TargetPlans_Id]
+					  ,[ReportingPeriod]
+					  ,Case when [ReportingPeriod] = '1617'  Then '1 April 2016 to 31 March 2017'
+							when [ReportingPeriod] = '1718'  Then '1 April 2017 to 31 March 2018'
```
[the remark this answers is not in the document]
Comment (sarmaevanidfe) [reply src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportPublicSectorDataToPL.sql]: Logic is amended based on [dbo].[DASCalendarMonth]

File: src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportPublicSectorDataToPL.sql (hunk 1 above)
[the remark this answers is not in the document]
Comment (sarmaevanidfe) [reply src/SFA.DAS.DATA.MANAGEMENT/StoredProcedure/ImportPublicSectorDataToPL.sql]: Truncate table is replaced with Delete for [AsData_PL].[PubSector_Report]

**Review by uhimabindu — APPROVED** [review]
Approved

**Pull request merged**